### ISSUE TITLE
Install Problems

### ISSUE THREAD

**Issue by dvreed77** [ContinuumIO/cdx#3]
So glad to see CDX is up!  Installed everything according to the README and ran

```
python cdxlocal.py
Starting server on port 5030...
[48879] 10 Sep 23:50:01.264 * Max number of open files set to 10032
```

Went to http://localhost:5030/cdx#cdx/, but just a black banner with 2 links (Continuum Data Explorer, Pop Out Plots), both lead to NotFound

Any ideas?

**Comment by pzwang** [comment]
Yep, cleaning up a few things right now.  Will be pushing some fixes and updating the README tonight.
